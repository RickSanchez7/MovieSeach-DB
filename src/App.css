* {
  box-sizing: border-box;
}
html {
  font-size: 16px;
  /* background-color: rgb(244, 244, 244); */
  height: 100vh;
  width: 100%;
  background: linear-gradient(
        to bottom,
        rgba(176, 210, 231, 0.9),
        rgba(228, 229, 230, 0.9)
      )
      no-repeat fixed,
    url('./assets/background.jpg') no-repeat center fixed;
  background-size: cover;
}
body {
  margin: 0;
  padding: 0;
  /* color: #333; */
  color: #1e90ff;
}

.container {
  /* margin: 0 auto; */
  max-width: 1000px;
  padding: 40px;
  /* text-align: center; */
}

.title {
  padding-top: 30px;
  font-size: 3.5rem;
  text-align: center;
}
