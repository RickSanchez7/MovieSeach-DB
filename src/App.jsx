import React from 'react';
import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
import Footer from './components/Footer/Footer';
// import Loading from './components/Loading/Loading';
import Navbar from './components/Navbar/Navbar';
import Home from './pages/Home';
import Movies from './pages/Movies';
import SingleShow from './pages/SingleShow';
import NotFound from './pages/NotFound';

// const Home = lazy(() => import('./pages/Home'));
// const Movies = lazy(() => import('./pages/Movies'));
// const SingleShow = lazy(() => import('./pages/SingleShow'));
// const NotFound = lazy(() => import('./pages/NotFound'));

const App = () => {
  return (
    <Router>
      <Navbar />
      <Switch>
        {/* <Suspense fallback={<Loading />}> */}
        <Route exact path="/" component={Home} />
        <Route path="/shows/:media/:id" component={SingleShow} />
        <Route path="/movies" component={Movies} />
        <Route path="/tv" component={Movies} />
        <Route component={NotFound} />
        {/* </Suspense> */}
      </Switch>
      <Footer />
    </Router>
  );
};

export default App;
