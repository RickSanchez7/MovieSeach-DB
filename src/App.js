import React from 'react';
import { Route, BrowserRouter as Router } from 'react-router-dom';
import './App.css';
import SearchMovies from './components/searchMovies/searchMovies';
import SingleMovie from './components/SingleMovie/SingleMovie.jsx';

class App extends React.Component {
  render() {
    return (
      <Router>
        <Route path='/' exact component={SearchMovies} />
        <Route path='/movies/:id' exact component={SingleMovie} />
      </Router>
    );
  }
}

export default App;
