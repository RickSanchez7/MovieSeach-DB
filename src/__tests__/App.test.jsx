import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

it('renders Navigation component', () => {
  expect(shallow(<App />)).toMatchSnapshot();
});
