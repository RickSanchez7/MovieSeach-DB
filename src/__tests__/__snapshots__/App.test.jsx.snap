// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navigation component 1`] = `ShallowWrapper {}`;
